tests/testthat/test-02_rdrop2-upload.R

```diff
@@ -0,0 +1,82 @@
+context("Testing drop_upload")
+library(dplyr)
+sprintf("Number of files on Dropbox", nrow(drop_dir()))
+
+
+# Test a basic file upload
+# -------------------------
+test_that("Test that basic file ops work correctly", {
+  skip_on_cran()
+
+  # This is a simple test to see if we can upload a csv file successfully
+
+  file_name <- paste0(uuid::UUIDgenerate(), "-file-ops-", ".csv")
+  write.csv(mtcars, file_name)
+  row_count <- nrow(mtcars)
+  print(file_name)
+  # Check to see if file uploads are successful
+  expect_message(drop_upload(file_name), "successfully at")
+  unlink(file_name)
+  drop_get(file_name)
+  y <- utils::read.csv(file_name)
+  server_row_count <- nrow(y)
+  # Make sure the downloaded csv has the same number of rows
+  expect_equal(row_count, server_row_count)
+  expect_message(drop_delete(file_name), "successfully deleted")
+  unlink(file_name)
+})
+
+# Test upload of an image
+# ------------------------
+
+test_that("Image upload works correctly", {
+  skip_on_cran()
+  # This test is to see if we can upload an image (a png in this case) and make sure that it maintains file integrity.
+  # File sizes are not perfectly equal down to the byte, hence the expect_equal with a tolerence.
```

Comment:
this strikes me as unexpected; we can't hash before/after and expect the same file?

Reply:
Done. The tests hash the files now